InventoryComponents/InventoryHUD/InventoryHUDRelicDisplay.gd

```diff
@@ -1,5 +1,5 @@
 extends Control
-class_name RelicDisplay
+class_name InventoryHUDRelicDisplay
```

Comment:
I think the previous name was clear while still being short

Reply:
I'm worried it's to vauge and that something down the line could use the same name